src/test/components/AppLocalizationProvider.test.js

```diff
@@ -9,51 +9,75 @@ import { AppLocalizationProvider } from 'firefox-profiler/components/app/AppLoca
 import { blankStore } from '../fixtures/stores';
 import { Provider } from 'react-redux';
 import {
-  getIsL10nFetching,
+  getl10nFetchingPhase,
   getLocalization,
 } from 'firefox-profiler/selectors/l10n';
 import { lazilyParsedBundles } from 'firefox-profiler/app-logic/l10n';
 import { requestL10n, receiveL10n } from 'firefox-profiler/actions/l10n';
-import { ReactLocalization } from '@fluent/react';
+import { ReactLocalization, Localized } from '@fluent/react';
+import { coerceMatchingShape } from '../../utils/flow';
+import { waitUntilState } from '../fixtures/utils';
 
 describe('AppLocalizationProvider', () => {
-  beforeEach(function() {
-    const response = (({
-      ok: true,
-      status: 200,
-      headers: {
-        get: () => 'text/plain',
-      },
-      text: () => Promise.resolve('resolved'),
-    }: any): Response);
-    window.fetch = jest.fn().mockResolvedValue(response);
-  });
+  function setupL10nFetch(language) {
+    jest.spyOn(window.navigator, 'languages', 'get').mockReturnValue(['en-US']);
```

Comment:
You should also change this mock return value and put the `language` directly instead of `'en-US'`.